runtimes/spiritnet/src/lib.rs

```diff
@@ -778,6 +798,7 @@ impl InstanceFilter<Call> for ProxyType {
 					// Excludes `ParachainSystem`
 					| Call::Preimage(..)
 					| Call::Proxy(..)
+					| Call::PublicCredentials(..)
```

Comment:
This is none deposit claiming. We actually need to filter out the reclaim deposit call?

Reply:
Good catch! https://github.com/KILTprotocol/kilt-node/pull/378/commits/9e43b3e89a336650d233cd22c38a1f6c1450c961